nss-cohort-40/bangazon-ecommerce-client-upperclasstwits
# Tb product list

# Description
Created a product list function that's linked to the route in application views.

Fixes 
## Type of change

# Testing Instructions
Login and click the "Products" tab on the navbar. Underneath the logo, you should see a list of the products in your database. Each product's title should be a hyperlink that leads you to its details view(currently not built, but the route is correct) 

# Checklist:
